packages/grid/examples/css-grid/src/pages/layouts.js

```diff
@@ -1,5 +1,5 @@
 import Head from 'next/head';
-import { Tile, Button, UnorderedList, ListItem } from 'carbon-components-react';
+// import { Tile, Button, UnorderedList, ListItem } from 'carbon-components-react';
```

Comment:
```suggestion
```

Might help for this diff to just remove any commented out code so we have a minimal working example without needing to come back and clean anything up in the future 👍